### PR TITLE
Add type hints to ModelValue.qname

### PR DESCRIPTION
**Pull request by ggravlingen**
#### Reason for change
More type hints are added.

#### Description of change
This PR:
- Adds type hints to `ModelValue.qname`
- Removes `type: ignore` referencing `ModelValue.qname`
- Fixes an error I made in the mypy-setting. It needs to be a .* to include all files. I believe the current file only includes the `__init__.py` file.

This PR will be rebased once #281 has been merged. That will remove the clutter of the white space being removed here. Also, I have not removed unused imports as I'm planning to do that in one sweep for all files following the completion of #281.

#### Steps to Test

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: pyproject.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ ignore_errors = true
 # when the library is converted, the above entry will be replaced (strict is demanded)
 [[tool.mypy.overrides]]
 module = [
-    'arelle.plugin.validate.ESEF',
+    'arelle.plugin.validate.ESEF.*',
```
Comment (ggravlingen) [inline pyproject.toml:26]: I made a mistake here. Do you want this, and related changes, to go in a separate PR? I believe the changes would be this line and the line edited in `Util.py`.

Comment (austinmatherne-wk) [reply pyproject.toml:26]: No, this is fine, it's all ESEF and type related. You can keep it together.

Comment (ggravlingen) [reply pyproject.toml:26]: OK, thanks! I'll leave it as a draft until the other one.

**Comment by ggravlingen** [comment]
I messed this branch up. Will start on a frech branch instead.
